### PR TITLE
Common: ModalAI setup instruction fix

### PR DESCRIPTION
**Pull request by rmackay9**
This is a minor fix to the VOXL camera configuration section based on feedback from @patrickpoirier51 and a couple of users who have reported issues.

This has been tested locally on my machine and seems fine.

### CODE REVIEW

File: common/source/docs/common-modalai-voxl.rst [hunk 1]
```diff
@@ -43,6 +43,7 @@ The minimum steps to get the camera working include
 
 - `Setup adb <https://docs.modalai.com/setup-adb/>`__ on an Ubuntu workstation
 - Connect the camera to the Ubuntu workstation, start and adb shell and `run voxl-configure-vision-px4 <https://docs.modalai.com/voxl-vision-px4-installation/>`__ (Note: you may simply press <enter> when asked for an IP address)
+- run "systemctl start voxl-mavlink-server" to configure the mavlink server to start automatically
```
Comment (khancyr) [inline common/source/docs/common-modalai-voxl.rst:46]: should be enable if you want it to always start on boot. Start will just start it once

Comment (patrickpoirier51) [reply common/source/docs/common-modalai-voxl.rst:46]: Yes , agree on that

Comment (rmackay9) [reply common/source/docs/common-modalai-voxl.rst:46]: @khancyr @patrickpoirier51,

Great, thanks very much for the correction.  I will change this to, "systemctl enable voxl-mavlink-server"

**Comment by rmackay9** [comment]
@khancyr @patrickpoirier51,

OK, fixed thanks.  Can you confirm this is this looks correct?

**Comment by patrickpoirier51** [comment]
Yes that is correct

**Comment by rmackay9** [comment]
@Hwurzburg 

I've gone ahead and merged this now, hope that's OK.

**Comment by Hwurzburg** [comment]
no problem....I was waiting for your requested review from  Pierre...I would have merged it then

**Pull request merged**